### DISCUSSION TITLE
Muu nimi

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha-translations#22] [Yleinen]
Olen joskus männä vuosina lisännyt asiakastiedoissa Muu nimi -kentän yhteyteen sulkeisiin varaustunnus-sanan, kun kenttää käytettiin varaustunnuksena. Koha-Suomessa on kuitenkin siirrytty tallentamaan varaustunnus asiakasmääreenä ja tuo kenttä halutaan ottaa käyttöön alkuperäisessä tarkoituksessaan. Poistin jo meidän omista käännöstiedostoista tiedon varaustunnuksesta, mutta olisiko ok, jos poistan sen myös yhteisön käännöksistä?

**Comment by hanhisalok** [comment]
Tägäsin Annelin jo korkeakoulujen Koha-Slackissä, mutta tiedoksi tännekin muille keskustelua lukeville, että korkeakoulukirjastot tosiaan käyttää edelleen muu nimi kenttää varaustunnukselle. Osalla numerosarja, osalla joku vapaavalintainen nimimerkki tms. Slackissa tosin ei vielä ottanut kukaan kantaa siihen, että voitaisiinko me tehdä asialle jotain, jotta tuon kentän käännös voisi jatkossa olla ns. oikein. Pitänee jatkaa vielä asiasta esim. meidän pääkäyttäjäkokouksessa.
